index.js

```diff
@@ -1,120 +0,0 @@
-const { Console } = require("console");
-const readline = require("readline");
-const readlineInterface = readline.createInterface(
-  process.stdin,
-  process.stdout
-);
-
-function ask(questionText) {
-  return new Promise((resolve, reject) => {
-    readlineInterface.question(questionText, resolve);
-  });
-}
-
-start();
-
-async function start() {
-  const welcomeMessage = `You awake with a sharp pain in the front of your head. Still slightly inebriated from the New Years party last night. \nYou look around to find yourself in a dungeon that is not your own. Figure it's about time to head out. \nIn the middle of the room you see a piece of paper and pen labelled CONTRACT TRACING. Please fill out your name to continue: `;
-  let name = await ask(welcomeMessage);
-  console.log(
-    `You write ${name} at the bottom of the page, barely legible. \nLooks like 2020 isn't over yet. `
-  );
-
-  //Asks you to begin the quest. Details are below
-  beginReply = await ask(`${name}, are you ready to begin your escape? `);
-  
-  
-// Class constructor for the rooms.
-  class Rooms {
-    constructor(room, description, direction,) {
-      this.room = room;
-      this.direction = direction;
-      this.description = description;
-    }
-    //Function to move between rooms
-    changeTo(direction) {
-      if (this.direction.includes(direction)) {
-        currentRoom = lookUpTable[direction];
```

Comment:
I love the use of saving what directions you can move to in the constructor and passing them as an array. Then using `this.direction.includes(direction)` to see if that is a valid room transfer. One comment about this is that because of your variables of `this.direction` and your `changeTo(direction)` both being the word direction, it can be confusing about which one is related to the Object and which is related to your variable being passed through the function argument.